fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -425,12 +425,12 @@ void FixedLagSmoother::transactionCallback(
 {
   // If this transaction occurs before the start time, just ignore it
   auto start_time = getStartTime();
-  const auto& min_time = transaction->minStamp();
-  if (started_ && min_time < start_time)
+  const auto max_time = transaction->maxStamp();
```

Comment:
Also note that we were doing `const auto& min_time`, which is a `const` reference to an attribute inside `transaction`, which is `std::move`d when it gets `insert`ed into the `pending_transactions_`, but still used later.

This wasn't an easy at the moment, but it could be a potential issue in the future if something changes the `std::move`d transaction. For that reason I'm taking a copy.

Reply:
:+1: Good catch.